modules/setTime.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable no-undef */
+export default function startTime() {
+  const localTime = luxon.DateTime.local();
+  const timeNow = document.querySelector('.time');
+  timeNow.innerText = `${localTime.toLocaleString(
+    luxon.DateTime.DATE_FULL,
+  )}, ${localTime.toLocaleString(luxon.DateTime.TIME_WITH_SECONDS)}`;
```

Comment:
Same here kindly replace this function with an arrow function. :+1: :smiley: :smiley: